packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -7,7 +7,7 @@ const jju = require('jju');
 /**
  *  @typedef {{root: string, name: string}} Options
  *  @typedef {{name: string, version: string, dependencies: {[key: string]: string}}} PackageJson
- *  @typedef {import("@nrwl/tao/src/shared/workspace").WorkspaceJsonConfiguration} WorkspaceJsonConfiguration
+ *  @typedef {import("@nrwl/devkit").WorkspaceJsonConfiguration} WorkspaceJsonConfiguration
```

Comment:
this type is used only internally so there is no need to specify this as dependency for this package. also looks like the lint rule is unaware to catch these kind of scenarios (where only types in vanilla JS are being used 🚨)